src/views/ConfusionMatrix/ConfusionMatrixViewData.ts

```diff
@@ -0,0 +1,34 @@
+import { validateObject } from "figurl"
+import { isArrayOf, isEqualTo, isNumber, isOneOf, isString } from "figurl/viewInterface/validateObject"
+
+export type ConfusionMatrixViewData = {
+    type: 'ConfusionMatrix'
+    sorting1UnitIds: (number | string)[]
+    sorting2UnitIds: (number | string)[]
+    unitEventCounts: {
```

Comment:
Does this refer to units on sorting 1, sorting 2, or both? If the sortings use the same naming scheme, how do you tell them apart?

Reply:
The unit IDs are assumed to be mutually exclusive between the sortings. So, A1, A2, A3, ..., B1, B2, B3, ...